gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -236,6 +236,10 @@ const CssVariablesPage = () => {
           <PropertyDocItem propertyVar="--nx-border-grid">
             The standard border style for <NxCode>nx-grid</NxCode> vertical borders.
           </PropertyDocItem>
+          <PropertyDocItem propertyVar="--nx-padding-offset-filter-clear-button">
+            The padding offset used to calculate the correct height of the clear button
+            in <NxCode>NxFilterInput</NxCode>.
+          </PropertyDocItem>
```

Comment:
![image](https://user-images.githubusercontent.com/24834551/231903870-c6d07280-dad6-46a6-baf9-696302e94ab7.png)
Looks like the gallery is having issues rendering this row. You may need to widen the first column.

Reply:
> ![image](https://user-images.githubusercontent.com/24834551/231903870-c6d07280-dad6-46a6-baf9-696302e94ab7.png) Looks like the gallery did not render this row correctly. You may need to widen the first column.

Removed: e7eeb6339c172b9ffa6ae94b9fb9cc35f2058bcb